registrations/models.py

```diff
@@ -146,6 +146,36 @@ def fire_language_metric(sender, instance, created, **kwargs):
         })
 
 
+@receiver(post_save, sender=Registration)
+def fire_source_metric(sender, instance, created, **kwargs):
+    """
+    Fires metrics for each source for each subscription, a sum metric for
+    the registrations over time, and a last metric for the total count.
+    """
+    from .tasks import fire_metric
+    if (created):
+        if "hw" in instance.source.authority:
+            source = 'hcw'
+            sources = ['hw_limited', 'hw_full']
+        else:
+            source = 'public'
+            sources = ['patient', 'advisor']
```

Comment:
I don't know which line the advisor uses. Maybe it's best to just keep the value as the `source` that's in the database, and let the dashboard designers decide what it means?